app/controllers/peer_reviews_controller.rb

```diff
@@ -136,14 +136,11 @@ def unassign(selected_reviewee_group_ids, reviewers_to_remove_from_reviewees_map
       reviewer_id_to_bool.each do |reviewer_id, dummy_value|
         reviewee_group = Grouping.find_by_id(reviewee_id)
         reviewer_group = Grouping.find_by_id(reviewer_id)
-        peer_review = reviewer_group.review_for(reviewee_group)
-        peer_review.destroy
+        PeerReview.delete_peer_review_between(reviewer_group, reviewee_group)
```

Comment:
After looking at this more, I think `unassign` and `assign` should be moved to be `PeerReview` class methods.